app/assets/stylesheets/styles/pages.scss

```diff
@@ -244,25 +258,25 @@
 
       .material-icons {
         background-color: #42a49a;
+        color: #fff;
         padding: 10px;
         position: relative;
+        transition-duration: 0.6s;
 
-        &::before {
-          content: "";
-          position: absolute;
-          width: 100%;
-          height: 100%;
-          background-color: #fff;
-          transform: rotate(8deg);
+        &:hover {
+          transform: translateY(-10px);
+          transition-duration: 0.6s;
         }
       }
     }
   }
 
   .why-use {
+    background-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.